src/components/CreateMessage.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Form, Input, InputNumber } from "antd";
+import { addMessage } from "../services/chain/apis/extrinsic";
+import { SchemaProps } from "../services/types";
+import { staticSchema } from "./RegisterSchema";
+
+
+
+const CreateMessage = (props: SchemaProps): JSX.Element => {
+    const [form] = Form.useForm();
+
+    const example_msg = { nickname:'omar',favorite_number: 6,favorite_restaurant:'Ramen Takeya'};
+
+    const handleSubmit =  async () => {
+
+        let isValid = validateMessage();
+        if (isValid) {
+            submitMessage();
+        };
+        form.resetFields();
```

Comment:
Do we want to reset if the validation fails?

Reply:
changed to behavior to simply write text 'message not valid' without resetting the fields